### PR TITLE
Feature/dockerised lambdas

### PR DESCRIPTION
**Pull request by romuaa**

### CODE REVIEW

File: backend/projects/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+## Testing locally
+
+Start the database and the lambda container
+
+```docker-compose -f docker-compose.lambda.yml up --build```
+
+Make a request to the function
+
+```curl -XPOST "http://localhost:9000/2015-03-31/functions/function/invocations" -d '{}'```
```
Comment (mikkilevon) [inline backend/projects/README.md:11]: BTW what is the /2015-03-31/ part of the URL?

Comment (romuaa) [reply backend/projects/README.md:11]: No idea why the path is like that. Another date did not work.
https://docs.aws.amazon.com/lambda/latest/dg/images-test.html

**Review by adamchabin — DISMISSED** [review]

File: backend/endpoints/log.txt [hunk 2]
```diff
@@ -0,0 +1,249 @@
+#1 [endpoints_dbmigration internal] load build definition from Dockerfile_migrations
```
Comment (mikkilevon) [inline backend/endpoints/log.txt:1]: I guess should be deleted?

**Review by mikkilevon — CHANGES_REQUESTED** [review]

**Review by CptPicard — APPROVED** [review]

File: backend/endpoints/log.txt (hunk 2 above)
Comment (romuaa) [reply backend/endpoints/log.txt:1]: I think it was already. Will do.

**Review by mikkilevon — APPROVED** [review]

**Review by CptPicard — APPROVED** [review]

**Pull request merged**